tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
 # Running REMIND locally
-Mika Pflüger (mika.pflueger@pik-potsdam.de)
+Mika Pflüger (mika.pflueger@pik-potsdam.de), Tonn Rüter (tonn.rueter@pik-potsdam.de)
 
-*Note:* This tutorial is a work in progress, not all steps needed are described, yet.
+*Note:* This approach of running REMIND is a work in progress, however successful REMIND runs on local machines have been conducted using this tutorial.
+
+## System Requirements
+
+- Installed python version must be `3.10.X` or older. To check your python version run `python --version` in a terminal of your choice
+- GAMS must be installed and the path to GAMS applications needs to be in your PATH environment variable
+- R version must be at least `4.2.X`. To check your R version run `R --version` in a terminal of your choice
+- On Windows we recommend to use PowerShell rather than the standard command prompt
+
+## Use Local Input Data
+
+**This is the supported way of input data handling**
+
+As long as we do not provide a curated collection of input data please obtain these files from the cluster:
+
+```bash
+rev6.606_62eff8f7_remind.tgz
+rev6.606_62eff8f7_validationremind.tgz
+CESparametersAndGDX_878ac5d69254efb4eba5c1fa39aba64000307bb1.tgz
+```
+
+The contents of these tar-archives are necessary to run REMIND locally. Since REMIND input data is constantly updated the up-to-date file names may vary, but will exhibit a similar pattern. The first two files can be found on the cluster at `/p/projects/rd3mod/inputdata/output`, the latter file in `/p/projects/remind/inputdata/CESparametersAndGDX`.
+
+Download & store them in a folder of your choice. The corresponding path is provided to REMIND via environment variable. Add to your local `~/.Renviron` file (in linux, found in your home directory, in windows, found at `C:\Users\<your windows username>\Documents\.Renviron`):
+
+```bash
+# REMIND data setup
+# If the input data is spread over multiple directories,
+# separate them with a : on linux, a ; on windows
+REMIND_repos_dirs="/my/first/path:/my/second/path"
+# On windows, it would look something like
+# REMIND_repos_dirs="C:\Users\<your windows username>\REMIND;D:\REMIND"
+# depending on your used file paths
+```
```

Comment:
As of #1457, user could also just run `make set-local-calibration` and then drop their files (both input data and CES gdxes) into `./calibration_results/`, which is included by default via [`.Rprofile`](https://github.com/remindmodel/remind/blob/9f6a2cb01f19b783dd8823ae0e6616c0c658962d/.Rprofile#L117-L123).
This might be easier if setting up just a single directory to run REMIND in.  For multiple directories a central location would be preferable, of course.

Reply:
This would be a third option, then. Tonn, can you try this approach and see if it works for you?  You can also check out [chapter 12 ](https://github.com/remindmodel/remind/blob/develop/tutorials/12_Calibrating_CES_Parameters.md#results)  to get a more detailed description of this approach.

Reply:
make is not available on Windows systems

Reply:
From https://github.com/remindmodel/remind/blob/develop/tutorials/11_ManagingRenv.md
> make is pre-installed except on Windows, run winget install gnuwin32.make to install it there

Does that work for you?

Reply:
If we add it as an option, it would be a third option (not replacing this option), and it can be described by referencing the description in the Results Section of the calibration chapter.